### PR TITLE
ffyaml: interpret null/empty value as empty string instead of error

### PR DESCRIPTION
**Pull request by peterbourgon**
Extension and slight reduction of #82

### CODE REVIEW

**Comment by peterbourgon** [comment]
@aramcodz does this suit your purposes?

File: fftest/vars.go [hunk 1]
```diff
@@ -99,6 +117,58 @@ func Compare(want, have *Vars) error {
 	return nil
 }
 
+func CompareTest(t *testing.T, want, have *Vars) error {
+	t.Helper()
+
```
Comment (aramcodz) [inline fftest/vars.go:122]: This new section of `CompareTest()`, adding the `WantParseErrorIs` section is a good refactoring.  👍

Comment (peterbourgon) [reply fftest/vars.go:122]: Oops!!! I meant to refactor that totally!!

Comment (peterbourgon) [reply fftest/vars.go:74]: Thanks for pointing it out.

**Comment by aramcodz** [comment]
> @aramcodz does this suit your purposes?

Yes, this changes works for me/us. Approved, Thanks!

**Review by aramcodz — APPROVED** [review]
Yup. This looks really good to me. 

Approved, Thanks!

**Comment by peterbourgon** [comment]
https://github.com/peterbourgon/ff/releases/tag/v3.1.0

**Comment by peterbourgon** [comment]
Sure thing. Sorry I was such a dingus for a bit there. Open source, huh? 😉

**Comment by aramcodz** [comment]
For the Lolz. All for the Lolz.

**Pull request merged**